sn_node/src/node/flow_ctrl/cmds.rs

```diff
@@ -216,6 +217,8 @@ impl Cmd {
 
             Comm(_) => 7,
 
```

Comment:
I think `AddToPendingQueries` always needs to be higher than `HandleValidSystemMsg` for node query respones, right? Can we note that here (it looks like it _is_ currently, doing this)

(An aside... I was wondering how we can explicitly link all the priority values across Cmds and the messages... I didn't have a good way. Not sure if you've any ideas, or if perhaps we should just look to declare _all_ of the prios in one place to avoid mistakes adjusting things here? )
```suggestion
// AddToPendingQueries needs to _always_ be handled before incoming query responses, otherwise we may have a response with no client recipients eg...
```

Reply:
Move Q to PR #1426